### PR TITLE
Update supported debian distributions

### PR DESCRIPTION
**Pull request by luozhenyu**
# Update supported debian distributions
> https://github.com/adoptium/installer/blob/master/linux/jdk/debian/src/packageTest/java/packaging/DebianFlavours.java#L32-L33
> Debian policy: Oldest, newest and development version.
> Ubuntu policy: Oldest LTS, newest LTS, and development version.

1. [Debian releases](https://wiki.debian.org/DebianReleases#Production_Releases)
    1. Add `debian:bookworm`
1. [Ubuntu releases](https://releases.ubuntu.com/)
    1. Remove `ubuntu:hirsute`
    1. Add `ubuntu:impish` and `ubuntu:kinetic`

### CODE REVIEW

File: linux/README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ export _JAVA_OPTIONS="-Xmx4g"
 - replace `<vendor>` with `temurin|dragonwell`
 
 ```shell
-export _JAVA_OPTIONS="-Xmx4g"
```
Comment (karianna) [inline linux/README.md:88]: Has the need for 4G gone away?  IIRC you need that amount of memory for the tool to run.

**Review by karianna — CHANGES_REQUESTED** [review]

File: linux/README.md (hunk 1 above)
Comment (luozhenyu) [reply linux/README.md]: Line 88 and line 90 are the same.
> ```
> export _JAVA_OPTIONS="-Xmx4g"
> export COMPOSE_DOCKER_CLI_BUILD=1
> export _JAVA_OPTIONS="-Xmx4g"
> ./gradlew clean packageJdkSuse checkJdkSuse --parallel -PPRODUCT=<vendor> -PPRODUCT_VERSION=<version>
> ```
https://github.com/adoptium/installer/blob/master/linux/README.md?plain=1#L88-L90

File: linux/README.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ export _JAVA_OPTIONS="-Xmx4g"
...
+export DOCKER_BUILDKIT=1
```
[the remark this answers is not in the document]
Comment (luozhenyu) [reply linux/README.md]: Enable the buildkit feature.

**Comment by luozhenyu** [comment]
@karianna I moved the documentation related codes to a separated PR #491

**Review by karianna — APPROVED** [review]

**Pull request merged**